### DISCUSSION TITLE
unable to use my student id for student pack

### DISCUSSION THREAD

**Discussion by The-Abhishek-Singh** [community/community#144563] [GitHub Education] [answered]
### Hi everyone,

i made a github account one year ago which is  Abhishek-2494 and purchase my student benefit on that account using my email which is abhisheks@ssipmt.com. but at that time I am not that active on git hub so I barely login my github account but after some time when i try to login my github account i was unable to login my account because i forgot that which authenticator app i used for my github and i can't find that and due to less knowledge i thought now i can't login my github account so i disconnect my number and email from that account and made a new github account which is Abhishek249494 . and try to purchase student benefit but github rejects my application and said my email which is abhi120730@gmail.com is already used to purchase student benefit on my another account which is  Abhishek-2494 but as i told you now i can't access that account and now i am using another github account which is Abhishek249494. i request you to please cancel my student benefit from my old account which is  Abhishek-2494 and you can even ban that account and allow me to get my student benefit on my new account which is Abhishek249494. my student email id is :- abhisheks@ssipmt.com

**Comment by davevad93** [comment, marked as answer]
Hi @Abhishek249494 ,

For account-related issues, it's recommended to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain the problem; however, note that it's not possible to transfer student benefits from one account to another, nevertheless contact support anyway to see if anything can be done.
